### PR TITLE
Fixing layout instability blog post

### LINKED ISSUE #1540: content: Fixing layout instability blog post
**Author**: @rviscomi
**Reviewer**: @kaycebasques

**Target publish date:** 2019-09-19

- [ ] Check this box if this is a hard deadline.
- [ ] Check this box if you'd like your reviewer to fix grammar and punctuation issues for you.

**Process**

Check out the [web.dev writing process](https://web.dev/handbook/quick-start/).

- [x] 1. I've done my [proposal](https://docs.google.com/document/d/1aadKmzxqoKkTO-3pRfnx5UDV5vNenYz58I8fVaUZr54/edit?usp=sharing)
- [x] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc
- [x] 4. Draft approved — _reviewer checks this_
- [ ] 5. I've submitted a pull request #<add PR number>
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by rviscomi**
Fixes #1540.

### CODE REVIEW

File: src/site/content/en/blog/fixing-layout-instability/index.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+In this example there was a single very tiny shift of 0.01% at 210ms.
+
+Knowing the time and severity of the shift is useful to help narrow down what could have caused the shift. Let's turn back to [WebPageTest](https://webpagetest.org) for a lab environment to do more testing.
+
+## Measuring layout shifts in WebPageTest
+
+Similar to measuring CLS in WebPageTest, measuring individual layout shifts will require a custom metric. Fortunately, the process is easier now that Chrome 77 is stable. The Layout Instability API is enabled by default, so you should be able to execute that JS snippet on any website within Chrome 77 and get results immediately. In WebPageTest, you can use the default Chrome browser and not have to worry about command line flags or using Canary.
+
+So let's modify that script to produce a custom metric for WebPageTest:
```
Comment (kaycebasques) [inline src/site/content/en/blog/fixing-layout-instability/index.md]:
```suggestion
So let's modify that script to produce a [custom metric for WebPageTest](https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/custom-metrics#TOC-Specifying-custom-metrics-at-test-time) called `[LayoutShifts]`:
```

On second look I think this link will help WPT newbies (like myself) follow along with the unfamiliar code syntax

**Comment by kaycebasques** [comment]
@rviscomi I suggested adding 2 links. There's a question for @robdodson which we'll need his input on before we can merge. So when you're done please assign this PR to Rob.

File: src/site/content/en/blog/fixing-layout-instability/index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+---
+title: Fixing layout instability
+subhead: A walkthrough of using WebPageTest to identify and fix layout instability issues.
+authors:
+  - rviscomi
+date: 2019-09-26
+hero: layout-comparison.png
```
[the remark this answers is not in the document]
Comment (robdodson) [reply src/site/content/en/blog/fixing-layout-instability/index.md]: mmm, no I don't think we should use this as a hero. It's pretty busy and gets cut off in the thumbnail view.

image

I think it's ok to just not have a hero, we have other posts that omit the hero image if there isn't a good relevant one to include. Another option is to look for something stock on https://unsplash.com.

**Comment by robdodson** [comment]
For all of the images in the article I'd suggest adding the `w-screenshot` class to them. This will give them a bit of extra bordering and make them seem a bit less like they're floating on the page.

File: src/site/content/en/blog/fixing-layout-instability/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kaycebasques) [reply src/site/content/en/blog/fixing-layout-instability/index.md]:
```suggestion
```

OK. Sounds like path of least resistance is to just nix the hero.

**Pull request merged**